### PR TITLE
Styling for Book Store

### PR DESCRIPTION
**Pull request by dheerajsachdeva**
- Design matches Zeplin's design

- Update progress functionality added - Not in requirement

### CODE REVIEW

File: src/components/style.css [hunk 1]
```diff
@@ -1,4 +1,292 @@
...
+  margin-right: 2rem;
+  opacity: 0.5;
+  font-family: Montserrat, Arial, Helvetica, sans-serif;
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
```
Comment (Zuhaib042) [inline src/components/style.css]: - [ ] This line is repeated multiple times in multiple selectors. it is against the best CSS practices. kindly consider adding a class 
```
.gray {
color: #121212;
} 
```
and then you can pass this class to any element you want. I am suggesting to follow best practices.

File: src/components/style.css [hunk 2]
```diff
@@ -1,4 +1,292 @@
...
+  height: 68.563rem;
+  margin: 0 2.5rem 4.063rem 0;
+  padding: 0 0 49.625rem;
+  border-radius: 4px;
+  box-shadow: 0 0 60px 0 rgba(0, 0, 0, 0.05);
+  background-color: var(--z);
+}
+
+.navbar {
+  display: flex;
```
Comment (Zuhaib042) [inline src/components/style.css]: - [ ]  This line is repeated multiple times in multiple selectors. it is against the best CSS practices. kindly consider adding a class
```
.flex {
display: flex;
}
```
and then you can pass this class to any element you want. I am suggesting to follow best practices.

File: src/components/style.css [hunk 3]
```diff
@@ -1,4 +1,292 @@
...
+  align-items: center;
+}
+
+.title {
+  font-family: RobotoSlab, Arial, Helvetica, sans-serif;
+  font-size: 1.375rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
```
Comment (Zuhaib042) [inline src/components/style.css]: - [ ] I think `normal` is the default value in most of the font-related styles so kindly consider removing these lines as they are redundant and following best practices we try to save the lines as much as possible.
- [ ] Kindly consider removing these lines in other selectors as well as I can see that you are styling this way in a lot of your selectors. If you need some specific `normal` style in some specific selector, then you can take `normal` value in that specific place.

File: src/components/style.css [hunk 4]
```diff
@@ -1,4 +1,292 @@
...
+  padding-top: 2rem;
+}
+
+.links {
+  padding: 1.438rem;
+  text-decoration: none;
+  margin-right: 2rem;
+  opacity: 0.5;
+  font-family: Montserrat, Arial, Helvetica, sans-serif;
+  font-size: 0.813rem;
```
Comment (Zuhaib042) [inline src/components/style.css:63]: - [ ] This line is repeated in multiple selectors. kindly consider making it a class and then give that class to any element you want as I suggested above. This way we can follow the best practices and DRY principle.

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @dheerajsachdeva , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✅ Identical Design
✅ No linters errors

## Required Changes ♻️
- [ ] Kindly consider adding an appropriate description of your project in the readme. right now the description is not accurate. it will make your readme look more professional.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/style.css (hunk 1 above)
Comment (dheerajsachdeva) [reply src/components/style.css]: Done. Thanks

File: src/components/style.css (hunk 2 above)
Comment (dheerajsachdeva) [reply src/components/style.css]: Done. Thanks.

File: src/components/style.css (hunk 3 above)
Comment (dheerajsachdeva) [reply src/components/style.css]: Removed. Thanks

File: src/components/style.css (hunk 4 above)
Comment (dheerajsachdeva) [reply src/components/style.css:63]: Font family added to body now. Thanks.

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢

Hi @dheerajsachdeva ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**